Automattic/jetpack
# Search: fix modal close button has dark background in light mode (Twenty Twenty One theme)

Fixes #19693

#### Changes proposed in this Pull Request:
Override `background-color` for close button for just Twenty Twenty-One theme. 

Fixed two issues,
- The selector of original CSS to fix this doesn't work, as there's no parent `.theme-twentytwentyone` class
- Corrected the fix and moved it to theme-tools module, where Jetpack modules have style fixes for official themes that could be loaded per theme, meaning if it's not Twenty Twenty-One theme, the fix wouldn't be load at all.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Go to a website with Jetpack Search subscription and Twenty Twenty-One theme enabled
- In customizer, set search overly colour to light
- Ensure the close button background is light but not dark
![image](https://user-images.githubusercontent.com/1425433/118729404-f8416e00-b889-11eb-8021-4fb40fc4cda7.png)
